src/app/multi-editor/multi-editor.component.html

```diff
@@ -1,3 +1,42 @@
-<ul *ngFor="let record of records">
-  {{record | json}}
-</ul>
+<img src="../assets/inspire_logo_hep.png" class='inspire-logo' />
+<div class="container">
+    <div class="row">
+      <div class="col-xs-8 col-xs-offset-2">
+        <div class="input-group item-group">
+        <div class="input-group-btn search-panel">
+          <div class="btn-group" dropdown>
+            <button dropdownToggle type="button" class="btn btn-default dropdown-toggle">
+                    	<span id="searchConcept">Filter by Collection</span> <span class="caret"></span>
+                    </button>
+            <ul *dropdownMenu class="dropdown-menu" role="menu">
+              <ul *ngFor="let collection of collections">
+                <li role="menuitem"><a class="dropdown-item" href="{{collection}}">{{collection}}</a></li>
+              </ul>
+            </ul>
+          </div>
+        </div>
+        <input type="text" [(ngModel)]="query" class="form-control" name="x" placeholder="Search term...">
+        <span class="input-group-btn">
+                    <button (click)="searchRecords()" class="btn btn-default" type="button"><span class="glyphicon glyphicon-search"></span></button>
+        </span>
+      </div>
+      <actions-template (onSubmit)="onSubmit($event)"></actions-template>
+      <ul class="list-unstyled">
+        <div style ="margin-left:73%">
+        <button class="btn-sm btn-success" type="button" (click)="selectAll()">Select All</button>
+        <button class="btn-sm btn-warning" type="button" (click)="deselectAll()">Deselect All</button>
+        </div>
+        <div *ngIf="totalRecords == 0"><h3>Sorry no results found</h3></div>
+        <li *ngFor="let record of records ; let count = index " class="list-group list-group-item">
+          <input type="checkbox" [checked]="(allSelected && checkedRecords.indexOf(record.control_number) === -1) || (allSelected === false && checkedRecords.indexOf(record.control_number)>-1)" (change)="addChecked(record.control_number)">
```

Comment:
remove `checked` and `change` and use `[(ngModel)]="selectionMap[record.control_number]"`

See the other comment on `ts` file.